bpf/proxy/syncer.go

```diff
@@ -17,6 +17,7 @@ package proxy
 import (
 	"context"
 	"fmt"
+	"math"
```

Comment:
I took a look at the surrounding code and I'm wondering if startupSync() is now out of date since it doesn't consider the new field and in general it seems to assume a 1-1 correspondence between services and map entries.  Would be worth adding a test that begins with some out-of-date source IPs in the dataplane and checks they get cleaned up.